nghttp/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+version: "1.41.0"
```

Comment:
`nghttp2ver.h` files says v1.22.0

Reply:
This was auto-generated file which had not been updated for long time. I have added update to this file now.